compiler+runtime/include/cpp/jank/jit/processor.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <boost/filesystem/path.hpp>
+#include <fmt/format.h>
```

Comment:
`fmt` is not permitted within normal headers. Please use our `util/string_builder` instead. `fmt` is too heavy and I have stripped it from every normal header and replaced it with a custom string builder which is much lighter to compile.

The commit for that is here: https://github.com/jank-lang/jank/commit/23aba5503bd5c3455511bf5e4ccc9e1b50e623ea

General usage:

```cpp
return err(util::string_builder{}("Failed to find the symbol: '")(name)("'")).release();
```

Reply:
Done